pages/dashboard.html

```diff
@@ -10,67 +10,79 @@
     <link rel="stylesheet" href="css/homepage.css">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
+    <script type="text/javascript" src="../js/dashboard.js"></script>
 </head>
 
 
-<body>
-    <!--NOTE: The use of inline CSS is a temporary fix. I am having difficulties with
-    Bootstrap overriding my custom external CSS. I will remove all inline CSS once I find
-    a solution to this issue.-->
-
+<body onload="getData()">
+    <!--Added Navigation Bar to the top of the Dashboard page-->
+    <nav class="navbar navbar-expand-sm bg-primary navbar-dark" id="main_navbar">
+        <div class="container-fluid">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a class="nav-link active" href="/dashboard">Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="/calendar">Calendar</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="/notebook">Notebooks</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="/settings">Settings</a>
```

Comment:
I thought we decided to get rid of the Settings?

Reply:
Yep, I just forgot to include it in this pull request. It'll be in the next one when I get the unit testing going later tonight